test/Project.toml

```
@@ -8,6 +8,7 @@ Trixi = "a7f1ee26-1774-49b1-8366-f1abc58fbfcb"
 
 [compat]
 Documenter = "0.27, 1"
+Downloads = "1"
```

```suggestion
Downloads = "1.7"
```
It should have the same version number as the Julia entry (in our main `Project.toml`) - see https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958